Editor/TestUtils/TestAsset.cs

```diff
@@ -54,6 +54,29 @@ public static TestAsset Save(UnityEngine.Object asset, string fileName)
             return tempAsset;
         }
 
+#if UNITY_2020_1_OR_NEWER
+        //SpriteAtlasAsset Save is not compatible with the AssetDatabase save
+        //Alternative function to create a TestAsset from a SpriteAtlas
+        public static TestAsset SaveSpriteAtlasAsset(SpriteAtlasAsset asset, string fileName)
+        {
+            var tempAsset = new TestAsset(fileName);
+            #if UNITY_2021_1_OR_NEWER
```

Comment:
The formatting is a bit weird here. I'd normally expect #if/else/endif statements to be fully left-aligned